src/System.Net.Security/src/System/Net/Security/SslStreamPal.Windows.cs

```diff
@@ -215,7 +215,7 @@ public static unsafe SecurityStatusPal EncryptMessage(SafeDeleteContext security
 
             // Copy the input into the output buffer to prepare for SCHANNEL's expectations
             input.Span.CopyTo(new Span<byte>(output, headerSize, input.Length));
-            
+
```

Comment:
Again, it'd be nice if files with no technical edits weren't part of the PR.

Reply:
will revert